README.md

```diff
@@ -146,6 +146,22 @@ class UserRegisterEmail {
 module.exports = UserRegisterEmail;
 ```
 
+The package has support for all events triggered in the bull, just add "on" and complete with the name of the event
+Ex: `onCompleted()`, `onActive()`, `onWaiting()` and etc.
+
+```js
+class UserRegisterEmail {
+  ...
+
+  onCompleted(job, result) {}
+  onActive(job) {}
+  ...
+}
+
+module.exports = UserRegisterEmail;
+
+```
+
```

Comment:
Nice!!
I think you could remove the old event description, your description is better!